pkg/proxy/ipvs/proxier.go

```diff
@@ -525,6 +528,23 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 	return append(mods, bmods...), nil
 }
 
+// GetKernelVersion returns currently running kernel version.
+func (handle *LinuxKernelHandler) GetKernelVersion() (*version.Version, error) {
+	kernelVersionFile := "/proc/sys/kernel/osrelease"
+	out, err := handle.executor.Command("cut", "-f1", "-d", " ", kernelVersionFile).CombinedOutput()
+	if err != nil {
+		return nil, fmt.Errorf("error getting os release kernel version: %v(%s)", err, out)
+	}
+	kernelVersionStr := strings.TrimSpace(string(out))
+	// parse kernel version
+	kernelVersion, err := version.ParseGeneric(kernelVersionStr)
+	if err != nil {
+		return nil, fmt.Errorf("error parsing kernel version: %v(%s)", err, kernelVersion)
```

Comment:
`"error parsing kernel version %q: %v", kernelVersion, err`?

Reply:
I moved this block from the old version, but you are right.